.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,2 +1,7 @@
 * Please describe the intent of your changes in a clear fashion.
 * Please make sure that, in the case of icon or mapping changes, you include images of these changes in the PR's description.
+
+[TODO]
+- Enter stuff that should be tested on the development server
+- These points will be presented ingame to the testers on the dev server
```

Comment:
Add an extra space after this line, so it wouldn't look weird if directly copy-pastad? The `[\TODO]` otherwise warps to the end of the list, not onto a new line, on the web browser.